client/lib/feed-post-store/post-batch-fetcher.js

```diff
@@ -88,8 +88,8 @@ assign( PostBatchFetcher.prototype, {
 					return;
 				}
 
-				containerId = +match[ 1 ];
-				postId = +match[ 2 ];
+				containerId = match[ 1 ];
```

Comment:
Though that's really a concern for the store, not for the batcher. Hrm. Maybe just leave this as is...